lib/brpoplpush/redis_script/config.rb

```diff
@@ -2,35 +2,49 @@
 
 module Brpoplpush
   module RedisScript
-    # ThreadSafe config exists to be able to document the config class without errors
-    ThreadSafeConfig = Concurrent::MutableStruct.new("ThreadSafeConfig",
```

Comment:
You may give a try to `Struct.new(:keyword_init)`

Reply:
The advantage of using a class is the type checks your are doing

Reply:
I prefer the class for that specific reason. I might change sidekiq unique jobs this way too. It is too easy to break the gem for this reason. I've spent too much time trying to figure out why something isn't working when it was just a wrong value in the config.